packages/frontend-2/components/viewer/limits/Dialog.vue

```diff
@@ -132,7 +132,9 @@ watch(dialogOpen, (value) => {
       type: props.limitType === 'version' ? 'version' : 'model',
       location: 'viewer',
       // eslint-disable-next-line camelcase
-      workspace_id: props.workspaceSlug
+      workspace_id: props.workspaceSlug,
+      // eslint-disable-next-line camelcase
+      limit_type: props.limitType
```

Comment:
Could you make this limitType? snake_case shouldnt be use for event property names, the workspace_id is the outlier in this case